idea-plugin/src/main/java/com/palantir/javaformat/intellij/InitialConfigurationStartupActivity.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google Inc. All Rights Reserved.
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
```

Comment:
Fair enough, the class is substantially different.

Reply:
I initially didn't notice this but I just wrote a new class and deleted old one